src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/activities/ActivitiesMigrationService.kt

```diff
@@ -53,7 +53,6 @@ class ActivitiesMigrationService(
   ): PageImpl<FindActiveActivityIdsResponse> =
     nomisApiService.getActivityIds(
       prisonId = migrationFilter.prisonId,
-      excludeProgramCodes = activitiesApiService.getActivityCategories(),
```

Comment:
The main change is that we no longer use the excluded program codes in nomis prisoner api, and so we no longer need the activity categories.